pom.xml

```diff
@@ -231,10 +231,17 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-javadoc-plugin</artifactId>
+          <version>3.1.0</version>
```

Comment:
Do these versions not come from the parent?

If not, we should fix that

Reply:
I pinned it because I want to have the control of it. It's mostly to support new version of java with the annoying linter.

Reply:
so you don't think we should be managing these versions in a parent, for any SmallRye project?

Reply:
Both :-)

The parent should give a recent enough version.
Projects can decide to override to tweak their build.

Reply:
fair enough